pkg/cidata/cidata.TEMPLATE.d/boot/40-install-containerd.sh

```diff
@@ -5,6 +5,28 @@ if [ "${LIMA_CIDATA_CONTAINERD_SYSTEM}" != 1 ] && [ "${LIMA_CIDATA_CONTAINERD_US
 	exit 0
 fi
 
+if [ "${LIMA_CIDATA_CONTAINERD_SYSTEM}" = 1 ] && [ -f /etc/alpine-release ]; then
+	# need to use the containerd packages
+	apk add runc containerd cni-plugins
+	rc-update add containerd default
+	rc-service containerd start
+
+	tar -xz -C /usr/local -f "${LIMA_CIDATA_MNT}"/nerdctl-full.tgz bin/buildctl bin/buildkitd
+	nohup buildkitd --oci-worker=false --containerd-worker=true >/var/log/buildkitd.log 2>&1 &
```

Comment:
Can we use openrc

Reply:
Sure, but I thought it was a bit long to include the scripts and config here so was saving it for the apk package...

https://github.com/lima-vm/alpine-lima/pull/36/files

Reply:
just added a placeholder for now